src/ansys/edb/core/hierarchy/layout_component.py

```diff
@@ -0,0 +1,63 @@
+"""Layout Component."""
+from ansys.api.edb.v1.layout_component_pb2 import ImportExportLayoutComponentMessage
+
+from ansys.edb.core.hierarchy.cell_instance import CellInstance
+from ansys.edb.core.inner.layout_obj import ObjBase
+from ansys.edb.core.session import LayoutComponentServiceStub, StubAccessor, StubType
+
+
+def _import_export_layout_component_msg(cell, output_aedb_comp_path):
```

Comment:
@drewm102 In this issue https://github.com/ansys/pyedb-core/issues/399,
_If the message is used multiple times but only in one module, create a private helper method in that module to generate those messages_
Do you think we should keep following this rule?

Reply:
@chenchienjacklin I think so. Do you agree?